data/tlg0565/tlg003/tlg0565.tlg003.1st1K-grc1.xml

```diff
@@ -49,21 +48,21 @@
 <sourceDesc>
 <listBibl>
  <biblStruct>
- <monogr>
-  <title>Sorani Gynaeciorum Libri IV De signis Fracturarum De Fasciis Vita Hippocratis Secundum Soranum</title>
-  <editor>
-	<persName>
-	 <name>Johannes Ilberg</name>
-	</persName>
-  </editor>
-  <author ref="urn:cts:greekLit:tlg0565">Soranus</author>
-  <imprint>
-	<publisher>Teubner</publisher>
-	<pubPlace>Leipzig</pubPlace>
-	<date>1927</date>
-  </imprint>
-  <biblScope unit="volume">4</biblScope>
- </monogr>
+  <monogr>
+   <title xml:lang="lat">Sorani Gynaeciorum Libri IV</title>
+   <title type="sub">De Signis Fracturarum De Fasciis Vita Hippocratis Secundum Soranum</title>
```

Comment:
<title type="sub"> needs language attribute

Reply:
Hey @lcerrato thanks for reviewing and I will make these changes. And this pull request as well as the other one you mentioned (tlg5075) are all on the same branch, I had created sub branches for them and actually did five or six different pull requests thinking somehow that would not create this mega pull request. Sorry. From now I'm just going to create a branch per textgroup even if it means there are more pull requests as they will be much easier to look at. Sorry for the all the work.